IdentityServer4.Contrib.RedisStore/Stores/PersistedGrantStore.cs

```diff
@@ -167,19 +167,19 @@ public async Task RemoveAllAsync(string subjectId, string clientId, string type)
             try
             {
                 var setKey = GetSetKey(subjectId, clientId, type);
-                var grantsKeys = await this.database.SetMembersAsync(setKey);
-                logger.LogDebug("removing {grantKeysCount} persisted grants from database for subject {subjectId}, clientId {clientId}, grantType {type}", grantsKeys.Count(), subjectId, clientId, type);
+                var grantsKeys = await _database.SetMembersAsync(setKey);
+                _logger.LogDebug("removing {grantKeysCount} persisted grants from database for subject {subjectId}, clientId {clientId}, grantType {type}", grantsKeys.Count(), subjectId, clientId, type);
                 if (!grantsKeys.Any()) return;
-                var transaction = this.database.CreateTransaction();
+                var transaction = _database.CreateTransaction();
                 transaction.KeyDeleteAsync(grantsKeys.Select(_ => (RedisKey)_.ToString()).Concat(new RedisKey[] { setKey }).ToArray());
                 transaction.SetRemoveAsync(GetSetKey(subjectId, clientId), grantsKeys);
                 transaction.SetRemoveAsync(GetSetKey(subjectId), grantsKeys);
                 await transaction.ExecuteAsync();
             }
             catch (Exception ex)
             {
-                logger.LogError(ex, "exception removing persisted grants from database for subject {subjectId}, clientId {clientId}, grantType {type}", subjectId, clientId, type);
-                throw ex;
+                _logger.LogError(ex, "exception removing persisted grants from database for subject {subjectId}, clientId {clientId}, grantType {type}", subjectId, clientId, type);
+                throw;
```

Comment:
thank you for fixing the exception throwing, I agree that we should capture the context via `throw;` not `throw ex;`
But why to change the coding style as well ?

Reply:
When rethrowing an exception, you should do it by simply calling throw; and not throw exc;, because the stack trace is reset with the second syntax, making debugging a lot harder.